modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -94,6 +103,59 @@ asmctl api create
 The API that was created is displayed.
 . Optionally, you can verify that an asset is created on Exchange and an API is created on API Manager.
 
+[[create-api-ASM-CLI-ConnAps]]
+=== Autocreate by Using Your Connected Apps Credentials
+
+If your users are managed by Anypoint Platform or an external IdP, use the Connected App credentials for authentication. Before you configure the autocreate feature, ensure that:
+
+* You xref:obtain-connected-apps-credentials.adoc[Obtain the client ID and client secret credentials] from Connected Apps
+* The ID of the group of assets on Exchange, for example, the organization ID
+* The client credentials has the:
+** Exchange Contributor permission for publishing the API to Exchange
+** Manage API Configuration permission for managing the API in API Manager
+
+To autocreate using Connected Apps, perform the following steps:
+
+. Using Anypoint Service Mesh, type the following command to create the API:
++
+[source,text,linenums]
+----
+asmctl api create
+--name=_Kubernetes API resource name_ \
+--namespace=_namespace name_ \
+--environmentId=_environment ID_ \
+--groupId=_(Optional) group ID of the asset, for example, organization ID_ \
+--assetId=_asset ID on Exchange_ \
+--version=_asset version of the instance on exchange_ \
+--instanceLabel=_label of the instance on API Manager_ \
+--specTags=_list of comma separated tags to be added to the asset on Exchange_ \
+--instanceTags=_list of comma separated tags to be added to the instance on API Manager_ \
+--clientId=_ID of the Anypoint Platform Connected Apps, using which to create the API instance_ \
+--clientSecret=_client secret of the Anypoint Platform Connected Apps, using which to create the API instance_
+----
++
+[%header%autowidth.spread]
```

Comment:
Do we REALLY need this table, given the text to the right of each equals symbol (=) in the syntax?
I'd omit it...or make it a supporting Reference topic....

Reply:
Yes because users miss reading it when given as part of the code. Done! Thanks!